css/index.css

```
@@ -1,3 +1,399 @@
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+}
 
+/* HTML5 display-role reset for older browsers */
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol, ul {
+  list-style: none;
+}
+
+blockquote, q {
+  quotes: none;
+}
+
+blockquote:before, blockquote:after, q:before, q:after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  --gray: rgba(0, 0, 0, 0.5);
+  --lightgray: rgba(127, 127, 127, 0.05);
+  --main-color: black;
+  --main-font: 'Titillium Web', sans-serif;
+  --main-width: 876px;
+  --top-nav-height: 5rem;
+  /* Media queries */
+  color: var(--main-color);
+  font-family: var(--main-font);
+  font-size: 16px;
+  height: 100%;
+}
+
+@media(max-width: 800px) {
+  html, body {
+    --main-width: 95vw;
+    --top-nav-height: 12rem;
+  }
+}
+
+@media(max-width: 500px) {
+  html, body {
+    --main-width: 100vw;
+  }
+}
+
+h1, h2, h3, h4, h5 {
+  font-family: 'Bangers', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+/*******/
+
+/* NAV */
+
+/*******/
+
+img {
+  max-width: 100%;
+}
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.top-nav-container {
+  align-items: center;
+  background: #fffffffa;
+  display: flex;
+  height: var(--top-nav-height);
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+@media (max-width: 500px) {
+  .top-nav-container {
+    position: relative;
+    height: auto;
+  }
+}
+
+.top-nav {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  width: var(--main-width);
+}
+
+@media (max-width: 800px) {
+  .top-nav {
+    align-self: flex-start;
+    flex-direction: column-reverse;
+  }
+}
+
+.top-nav nav {
+  display: flex;
+  justify-content: space-around;
+  width: calc(100% - 14rem);
+}
+
+@media (max-width: 800px) {
+  .top-nav nav {
+    width: 100%;
+    justify-content: space-between;
+  }
+}
+
+@media (max-width: 500px) {
+  .top-nav nav {
+    flex-direction: column;
+    font-size: 1.2rem;
+  }
+}
+
+@media (max-width: 800px) {
+  .top-nav .logo {
+    margin: 2rem 0;
+  }
+}
+
+@media (max-width: 500px) {
+  .top-nav .logo {
+    padding: 1rem 0 1.5rem;
+  }
+}
+
+.top-nav nav a {
+  color: var(--gray);
+  padding: 0.5rem;
+  text-decoration: none;
+}
+
+.top-nav nav a:hover {
+  color: var(--main-color);
+  text-shadow: white 0 0 2px;
+}
+
+@media (max-width: 500px) {
+  .top-nav nav a {
+    text-align: center;
+    padding: 1.26rem 0;
+    border-top: 1px solid var(--gray);
+    border-bottom: 1px solid var(--gray);
+  }
+  .top-nav nav a+a {
```

I don't see the purpose of `+a` here
```suggestion
  .top-nav nav a { 
```

---

Just trying out that query selector, it only selects those that have an `a` directly before it, so essentially all but the first-of-type. I agree it’s not really necessary, but I was just experimenting to see how it worked 🙃